base/applications/cleanmgr/dialog.c

```diff
@@ -0,0 +1,360 @@
+/*
+ * PROJECT:         ReactOS Disk Cleanup
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Dialog Procs
+ * COPYRIGHT:       Copyright 2020 Arnav Bhatt (arnavbhatt288 at gmail dot com)
+ */
+ 
+#include "resource.h"
+#include "precomp.h"
+
+#define CX_BITMAP 18
+#define CY_BITMAP 18
+
+DIRSIZE sz;
+DLG_VAR dv;
+WCHAR_VAR wcv;
+BOOL_VAR bv;
+
+BOOL CALLBACK StartDlgProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+	WCHAR LogicalDrives[MAX_PATH] = { 0 };
+	int ItemIndex = 0;
+	static HBITMAP hbmDrive;
+	static HBITMAP hbmMask;
+	DWORD dwIndex = 0;
+
+	bv.sysDrive = FALSE;
+	
+	DWORD drives = GetLogicalDriveStringsW(MAX_PATH, LogicalDrives);
+	if (drives == 0)
+	{
+		MessageBoxW(NULL, L"GetLogicalDriveStringsW() failed!", L"Error", MB_OK | MB_ICONERROR);
+		return FALSE;
+	}
+	
+	switch(Message)
+    {
+		case WM_INITDIALOG:
+
+			hbmDrive = LoadBitmapW(dv.hInst, MAKEINTRESOURCE(IDB_DRIVE));
+			hbmMask = LoadBitmapW(dv.hInst, MAKEINTRESOURCE(IDB_DRIVE));
```

Comment:
Mask image is same as the normal image. Something is wrong.

Reply:
You can omit the mask image.

Reply:
@katahiromz Isn't mask image is like an original image but the background is white and the "actual" image is white for transparency?